LAYOUT.md

```diff
@@ -74,6 +73,11 @@ HTML that are used in several page layouts.  These directories have
 underscores at the start of their names to be consistent with Jekyll's
 naming conventions, but the files they contain are for Pandoc.
 
+Most authors should not need to modify any of the support files themselves.
+The only exception is the `_includes/footer.html` file linking the "Source"
+button to [https://github.com/swcarpentry/lesson-template][swc-lesson-template].
```

Comment:
On Wed, Dec 02, 2015 at 02:15:33PM -0800, Alexander Konovalov wrote:
> I've updated the PR. 

b31c9a82b8 looks good to me.